k8s/helm/templates/locust-deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}-locust-slave
```

Comment:
Can we remove all the references to `slave` since this is the distributed mode anymore?